tests/fixtures/joint.py

```diff
@@ -43,15 +49,46 @@ class JointFixture:
     mongodb: MongoDbFixture
     rest_client: AsyncTestClient
 
+    def remove_db_data(self) -> None:
+        """Delete everything in the database to start from a clean slate"""
+        self.mongodb.empty_collections()
 
-@pytest_asyncio.fixture
-async def joint_fixture(
-    populated_mongodb_fixture: MongoDbFixture,  # noqa: F811
+    def load_test_data(self) -> None:
+        """Populate a collection for each file in test_data"""
+        filename_pattern = re.compile(r"/(\w+)\.json")
+        for filename in glob.glob("tests/fixtures/test_data/*.json"):
+            match_obj = re.search(filename_pattern, filename)
+            if match_obj:
+                collection_name = match_obj.groups()[0]
```

Comment:
```suggestion
                collection_name = match_obj.group(1)
```

Reply:
Changed in [b03a67b](https://github.com/ghga-de/mass/pull/9/commits/b03a67b36c54a7f0460d50a02bc693029d2a9a95)